.github/workflows/build_and_test.yaml

```diff
@@ -65,14 +64,14 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v2
         with:
-          path: src
           ref: ${{ github.event.inputs.ref || github.ref }}
 
       - name: Build Player
-        uses: ./src/.github/workflows/custom-actions/build-player
+        shell: bash
+        run: src/build/build.py
```

Comment:
You can drop src here, and below, as well.  `actions/checkout` with a path should result in the working directory being the source code root.